content/BPIP-4.md

```diff
@@ -0,0 +1,220 @@
+---
+bpip: 4
+title:  Support price discovery 
+discussions-to: https://github.com/bosonprotocol/BPIPs/discussions/9
+status: Draft 
+created: 2023-01-25
+---
+
+## Abstract
+
+This proposal defines an Interface that extends the protocol to enable price discovery. The changes proposed will allow Sellers to pick their price discovery method of choice so that they can leverage external price discovery mechanisms. .
```

Comment:
```suggestion
This proposal defines an Interface that extends the protocol to enable Price Discovery. The changes proposed will allow Sellers to pick their price discovery method of choice so that they can leverage external price discovery mechanisms.
```

Reply:
done